server/server.js

```diff
@@ -139,8 +139,24 @@ app.get(`${apiBaseUrl}/employees`,
 // Create employee records from uploaded XLSX file
 app.post(`${apiBaseUrl}/employees`,
   keycloak.protect(allowRoles('maximus')),
-  multer().single('file'),
+  multer({
+    fileFilter: (req, file, cb) => {
+      if (file.fieldname !== 'file') {
+        req.fileError = 'Invalid field name.';
+        return cb(null, true);
```

Comment:
Shouldn't the file be rejected here (`cb(null, false)`)?

Reply:
Oops, good catch!